sdk/eventhub/event-hubs/src/eventDataAdapter.ts

```diff
@@ -10,11 +10,11 @@ import { EventData } from "./eventData";
  *
  * @hidden
  */
-export interface MessageWithMetadata {
+export interface MessageContent {
   /**
    * The message's binary data
    */
-  body: Uint8Array;
+  data: Uint8Array;
```

Comment:
We discussed this for .NET and chose to `data` to avoid having the situation where you have code like:
``` content.Content = "content"```

Reply:
gotcha. JS also interacts with the actual "MessageContent" object (unlike Python and like .NET), and so would use `data` to avoid `content.content` situation?

Reply:
@swathipil in the popular cases no but the customer can always provide their own adapter to convert from MessageContent to their own message type so there is a chance to interact with the data field directly.